### PR TITLE
Allow setting attributes on the picture tag

### PR DESCRIPTION
**Pull request by 0xJoKe**
TL;DR These changes would allow passing attributes to the `picture` tag by prefixing attributes with `picture:`.

Sometimes it is necessary to style the picture tag, which is currently not possible using Tailwind CSS (the CSS Framework used by Statamic itself). While I understand that the goal is to keep this addon as simple as possible, I do think this should be core functionality. As far as I can tell publishing the views would not help, since the parameters would not get passed through ~and wrapping the picture tag in a styled div does not work for all instances~ (← could not reproduce).

Sticking with the notation used for the `glide` parameters, I decided to allow this by passing parameters prefixed with `picture:`.

### CODE REVIEW

**Comment by riasvdv** [comment]
> wrapping the picture tag in a styled div does not work for all instances.

When does this not work? I've never come across an instance where I had to style the <picture> tag specifically.

**Comment by 0xJoKe** [comment]
I just spent an hour trying to reproduce my problem from yesterday and I failed miserably (which is a good thing I guess?) 😅 

But I do still think, that avoiding unnecessary markup is a good thing.

**Comment by riasvdv** [comment]
But this moves the markup inside the responsive tag, I personally think a wrapping div or other element with attributes is a lot more clear than all those attributes contained in the tag.

I'm closing this for now, if you ever hit your problem again I'll consider adding this.
